public/src/js/constants/article-meta-fields.js

```diff
@@ -64,6 +64,30 @@ export default Object.freeze([
         type: 'text'
     },
     {
+        key: 'imageSource',
+        editable: true,
+        dropImage: true,
+        omitForSupporting: true,
+        visibleWhen: 'imageReplace',
+        label: 'replacement image URL',
+        validator: {
+            params: {
+                src: 'imageSrc',
+                width: 'imageSrcWidth',
+                height: 'imageSrcHeight',
+                origin: 'imageSrcOrigin',
+                imageSource: 'imageSource',
+                options: {
+                    maxWidth: 1000,
```

Comment:
But changing it to pick the biggest wouldn't automatically choose correctly `/master/` (there is another crappier crop of same width next to `/master/`, we should choose it only if the `/master/` is not there - like with old assets). Sorry if preaching to the choir!

BTW - now I know why I have to have GIFs over 400px wide while trying to satisfy Grant's 100KB size limit (a line below). Not that we want more GIFs. Or JPGs below 400px wide... I don't!